tests/test_models.py

```diff
@@ -1,7 +1,9 @@
 from osier import DispatchModel
-from osier import Technology
+from osier import Technology, ThermalTechnology, StorageTechnology
```

Comment:
It looks like you only test the `ThermalTechnology` and `StorageTechnology`. What about the `RampingTechnology` or the `RampingStorageTechnology`?

Reply:
Themal technology inherits from ramping technology, so that would duplicate a test. The latter class hasn't been used yet but also doesn't implement any features that haven't already been tested. I can add a test for it next week if you think it's necessary.

Reply:
Okay, I removed `RampingStorageTechnology` from the PR.